packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -204,7 +204,7 @@ - (void)startVideoRecordingAtPath:(NSString *)path result:(FlutterResult)result;
 - (void)stopVideoRecordingWithResult:(FlutterResult)result;
 - (void)startImageStreamWithMessenger:(NSObject<FlutterBinaryMessenger> *)messenger;
 - (void)stopImageStream;
-- (void)captureToFile:(NSString *)filename result:(FlutterResult)result;
+- (void)captureToFile:(NSString *)filename result:(FlutterResult)result API_AVAILABLE(ios(10));
```

Comment:
All the method declarations can be removed from this interface, it just needs the properties.  `API_AVAILABLE(ios(10.0))` can go in the implementation:

```obj-c
- (void)captureToFile:(NSString *)path result:(FlutterResult)result API_AVAILABLE(ios(10.0)) {
```